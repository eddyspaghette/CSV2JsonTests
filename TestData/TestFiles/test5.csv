1,100,200,300,400,500
2,101,201,301,401,501
3,102,202,302,,502
4,103,203,303,403,503
24,123,223,323,423,523
6,105,205,305,405,505
7,106,206,306,406,506
,107,207,307,407,507
,,208,,408,508
10,109,,309,409,509
11,110,,,410,510
12,111,211,,411,511
13,112,212,312,412,512
2,101,201,301,401,501
15,114,214,314,414,514
16,115,215,315,415,515
17,116,216,316,416,516
18,,217,317,417,517
19,118,218,318,418,518
10,109,209,309,409,509
21,120,220,320,420,520
22,121,221,321,421,521
23,122,222,322,422,522
24,123,223,323,423,523
