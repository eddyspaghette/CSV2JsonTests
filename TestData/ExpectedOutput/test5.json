[
 {
   "FIELD1": 1,
   "FIELD2": 100,
   "FIELD3": 200,
   "FIELD4": 300,
   "FIELD5": 400,
   "FIELD6": 500
 },
 {
   "FIELD1": 2,
   "FIELD2": 101,
   "FIELD3": 201,
   "FIELD4": 301,
   "FIELD5": 401,
   "FIELD6": 501
 },
 {
   "FIELD1": 3,
   "FIELD2": 102,
   "FIELD3": 202,
   "FIELD4": 302,
   "FIELD5": null,
   "FIELD6": 502
 },
 {
   "FIELD1": 4,
   "FIELD2": 103,
   "FIELD3": 203,
   "FIELD4": 303,
   "FIELD5": 403,
   "FIELD6": 503
 },
 {
   "FIELD1": 24,
   "FIELD2": 123,
   "FIELD3": 223,
   "FIELD4": 323,
   "FIELD5": 423,
   "FIELD6": 523
 },
 {
   "FIELD1": 6,
   "FIELD2": 105,
   "FIELD3": 205,
   "FIELD4": 305,
   "FIELD5": 405,
   "FIELD6": 505
 },
 {
   "FIELD1": 7,
   "FIELD2": 106,
   "FIELD3": 206,
   "FIELD4": 306,
   "FIELD5": 406,
   "FIELD6": 506
 },
 {
   "FIELD1": null,
   "FIELD2": 107,
   "FIELD3": 207,
   "FIELD4": 307,
   "FIELD5": 407,
   "FIELD6": 507
 },
 {
   "FIELD1": null,
   "FIELD2": null,
   "FIELD3": 208,
   "FIELD4": null,
   "FIELD5": 408,
   "FIELD6": 508
 },
 {
   "FIELD1": 10,
   "FIELD2": 109,
   "FIELD3": null,
   "FIELD4": 309,
   "FIELD5": 409,
   "FIELD6": 509
 },
 {
   "FIELD1": 11,
   "FIELD2": 110,
   "FIELD3": null,
   "FIELD4": null,
   "FIELD5": 410,
   "FIELD6": 510
 },
 {
   "FIELD1": 12,
   "FIELD2": 111,
   "FIELD3": 211,
   "FIELD4": null,
   "FIELD5": 411,
   "FIELD6": 511
 },
 {
   "FIELD1": 13,
   "FIELD2": 112,
   "FIELD3": 212,
   "FIELD4": 312,
   "FIELD5": 412,
   "FIELD6": 512
 },
 {
   "FIELD1": 2,
   "FIELD2": 101,
   "FIELD3": 201,
   "FIELD4": 301,
   "FIELD5": 401,
   "FIELD6": 501
 },
 {
   "FIELD1": 15,
   "FIELD2": 114,
   "FIELD3": 214,
   "FIELD4": 314,
   "FIELD5": 414,
   "FIELD6": 514
 },
 {
   "FIELD1": 16,
   "FIELD2": 115,
   "FIELD3": 215,
   "FIELD4": 315,
   "FIELD5": 415,
   "FIELD6": 515
 },
 {
   "FIELD1": 17,
   "FIELD2": 116,
   "FIELD3": 216,
   "FIELD4": 316,
   "FIELD5": 416,
   "FIELD6": 516
 },
 {
   "FIELD1": 18,
   "FIELD2": null,
   "FIELD3": 217,
   "FIELD4": 317,
   "FIELD5": 417,
   "FIELD6": 517
 },
 {
   "FIELD1": 19,
   "FIELD2": 118,
   "FIELD3": 218,
   "FIELD4": 318,
   "FIELD5": 418,
   "FIELD6": 518
 },
 {
   "FIELD1": 10,
   "FIELD2": 109,
   "FIELD3": 209,
   "FIELD4": 309,
   "FIELD5": 409,
   "FIELD6": 509
 },
 {
   "FIELD1": 21,
   "FIELD2": 120,
   "FIELD3": 220,
   "FIELD4": 320,
   "FIELD5": 420,
   "FIELD6": 520
 },
 {
   "FIELD1": 22,
   "FIELD2": 121,
   "FIELD3": 221,
   "FIELD4": 321,
   "FIELD5": 421,
   "FIELD6": 521
 },
 {
   "FIELD1": 23,
   "FIELD2": 122,
   "FIELD3": 222,
   "FIELD4": 322,
   "FIELD5": 422,
   "FIELD6": 522
 },
 {
   "FIELD1": 24,
   "FIELD2": 123,
   "FIELD3": 223,
   "FIELD4": 323,
   "FIELD5": 423,
   "FIELD6": 523
 }
]